bcipy/helpers/tests/test_triggers.py

```diff
@@ -34,14 +45,16 @@ def sample_raw_data(trigger_seq: List[Tuple[str, str]] = [],
 
     # Mock the raw_data file
     sep = '\r\n'
-    meta = sep.join(['daq_type,TestStream', 'sample_rate,300'])
-    header = 'timestamp,c1,c2,c3,TRG'
+    meta = sep.join([f'daq_type,{daq_type}', 'sample_rate,{sample_rate}'])
+    header = 'timestamp,' + ','.join(ch_names) + ',TRG'
 
     data = []
     for i in range(1000):
         timestamp = i + 10.0
-        channel_data = [str(random.uniform(-1000, 1000)) for _ in range(3)]
-        trg = triggers_by_time.get(timestamp, NONE_VALUE)
+        channel_data = [
+            str(random.uniform(-1000, 1000)) for _ in range(len(ch_names))
```

Comment:
I want to standardize how we mock our data - I think we have a random data generator function we can use / extend for your cases.

Reply:
Yes, good point. I'll update this to use the acquisition module.